### ISSUE TITLE
old_doc_file for partial build not defined

### ISSUE THREAD

**Issue by paulobrecht** [RGLab/DataPackageR#15]
I'm running a partial build of a package from the parent directory, and old_doc_file is getting a length 0 value, which causes the code to crash.

```
Output created: /Volumes/pobrecht/GITprojects/Uberla465/inst/extdata/Logfiles/preprocess_bama.html
Saving challenge as Uberla465_challenge.
Found 6 data objects in file datasets_bama_only.R
Processed data sets match existing data sets at version 0.1.18
Saving to data
Error in if (file.exists(old_doc_file)) { : argument is of length zero
```

Here's the relevant code from processData.R:

```
old_doc_file = list.files(here("R"),full.names = TRUE,paste0(pkg_description$Package,".R"))
if(file.exists(old_doc_file)){
             old_docs = .parseDocumentation(old_doc_file)
              merged_docs = .mergeDocumentation(old = old_docs, new = doc_parsed)
              save_docs = do.call(c,merged_docs)
            }
```

Seems to be an issue with `here("R")`. I am building the package from the parent directory, so running `buildDataSetPackage("packagename")` rather than running `buildDataSetPackage()` from within the main dir of the package. The path defined by `here()` is defined when the `here` package is loaded, so maybe `here` is being loaded before the `setwd()` into the data-dir directory? Everything does seem to work when I run `buildDataSetPackage()` from within the package dir.

**Comment by gfinak** [comment]
Here's the piece that's not behaving as expected:

https://github.com/RGLab/DataPackageR/blob/cc5f3711193967e3474dc723123482c5ec7f6d1f/R/processData.R#L67

**Comment by gfinak** [comment]
This is fixed in version 1.12.0 as I've moved away from the `here` package in favor of other mechanisms.
